config/mcp_config.py

```
@@ -0,0 +1,75 @@
+            except Exception:
+                pass
+        return self.default_config
+    
+    def get_endpoints(self) -> Dict[str, str]:
+        """Get MCP server endpoints"""
+        return self.get_config().get("endpoints", self.default_config["endpoints"])
+    
+    def save_config(self, config: Dict[str, Any]):
+        """Save configuration"""
+        self.config_path.parent.mkdir(parents=True, exist_ok=True)
+        with open(self.config_path, 'w') as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)